<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_re</class>
 <widget class="QDialog" name="dialog_re">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>424</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Sửa giá trị trực tiếp</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Get value</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Set value</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Delete value</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Create key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Delete key</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="placeholderText">
       <string>Đường dẫn</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="key_value">
         <property name="placeholderText">
          <string>Name Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="value">
         <property name="placeholderText">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>String</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Binary</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DWORD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>QWORD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Muti-String</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Expandable-String</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="noti">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btn_send1">
         <property name="text">
          <string>Gửi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_xoa">
         <property name="text">
          <string>Xóa</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_browser">
      <property name="text">
       <string>Browser</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_send">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>Gửi nội dung</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
